themes/classic/modules/contactform/views/templates/widget/contactform.tpl

```diff
@@ -62,7 +62,7 @@
               class="form-control"
               name="from"
               type="email"
-              value="{$contact.email}"
+              value="{$customer.email}"
```

Comment:
As `contact` is sent by the module, I would rather keep it instead of relying of a variable from the core which could be removed.